packages/dashboard/package.json

```diff
@@ -22,60 +22,56 @@
   ],
   "scripts": {
```

Comment:
if you want style/lint enforcement, should you setup husky here with eslint configured correctly? But this is hard in a mono repo.

Reply:
Husky hook (to lint on precommit) _does_ run.